packages/react-components/src/templates/__tests__/TeamCreateOutputPage.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/react';
+import { ResearchOutput } from '@asap-hub/model';
+import { createResearchOutputResponse } from '@asap-hub/fixtures';
+import userEvent from '@testing-library/user-event';
+import TeamCreateOutputPage from '../TeamCreateOutputPage';
+
+describe('TeamCreateOutputPage', () => {
+  test('renders the research output type in the header', () => {
```

Comment:
The rest of the FE uses it rather than test

Reply:
Thanks! Done.

Reply:
thanks, done